notebooks/official/model_monitoring/get_started_with_model_monitoring_automl.ipynb

```diff
@@ -1029,9 +998,9 @@
       "source": [
         "### Logging sampled requests\n",
         "\n",
-        "Once the monitoring service has started, the sampled prediction requests will be logged to Cloud Storage. On the next monitoring interval, the sampled predictions are then copied over to the BigQuery logging table. Once the entries are in the BigQuery table, the monitoring service will analyze the sampled data.\n",
+        "Once the monitoring service has started, the sampled prediction requests are logged to Cloud Storage. On the next monitoring interval, the sampled predictions are copied to the BigQuery logging table. Once the entries are logged, the monitoring service analyzes the sampled data.\n",
         "\n",
-        "Next, you wait for the first logged entres to appear in the BigQuery table used for logging prediction samples. Since you sent 1000 prediction requests, with 50% sampling, you should see around 500 entries."
+        "Next, you wait for the first logged entries to appear in the BigQuery logging table for prediction samples. Since you sent 1000 prediction requests, with 50% sampling, you should see around 500 entries.s.g, you should see around 500 entries."
```

Comment:
Think there's repetition:

- you should see around 500 entries.s.g, you should see around 500 entries

Shouldn't this be:

- you should see around 500 entries.